libs/docs/src/docs/no-version/quickstart-odo.md

```diff
@@ -0,0 +1,219 @@
+---
+title: "Quick start with odo v3"
+description: "Quick start with odo v3"
+---
+
+This guide will run through creating a simple hello world devfile project using `odo` version 3. The purpose of this guide is to provide a practical introduction to those just starting out with devfiles.
+
+## Prerequisites
+
+- `odo` v3
+- `kubectl` or `oc`
```

Comment:
Why is `kubectl` or `oc` as a prerequisite? Odo doesn't require either of those and this guide doesn't seem to use them as well.

Reply:
> Why is `kubectl` or `oc` as a prerequisite? Odo doesn't require either of those and this guide doesn't seem to use them as well.

Good catch, I missed that `odo` has its own `login` and `logout` commands despite sharing the same login session that `kubectl` and `oc` uses.